### ISSUE TITLE
Onscreen keyboard doesn't work on Ubuntu

### ISSUE THREAD

**Issue by igor2x** [pwsafe/pwsafe#1103]
1. I installed Password Safe v1.18.2 on Ubuntu 23.10. On Safe Combination Entry screen when I click on Onscreen keyboard icon error is returned to install xvkbd package.
2.  I installed package: `sudo apt install xvkbd`
3. Restarted Password Safe and clicked on Onscreen keyboard icon.
4. Now onscreen keyboard is displayed. But I have no idea how this should suppose to work or something is broken.

ADDITIONAL INFO:
a) On Password Safe on Windows there is the following onscreen keyboard:
![image](https://github.com/pwsafe/pwsafe/assets/10895030/86bc2562-f498-4239-9fee-c6639fc15618)
b) I use keyboard to click on individual letter and then when I am finished, then click on Insert button on onscreen keyboard and password is typed in.

I can't find any similar Insert button on Linux. I also clicked on Return and/or Enter and in this case string disappears in onscreen keyboard program, but nothing appears in Password Safe. I don't know which of the onscreen key is responsible to enter string into target program like Password Safe. Any help?

If this is of any use, I use Wayland as display server, because on my laptop when I select X-server at Ubuntu logon, then whole Ubuntu freezes. In my case laptop only works on Wayland, so can't test if this works on X-server or not.

**Comment by ronys** [comment]
Yes, I see that xvkbd has trouble typing into terminal windows as well, but works with vscode.

Needs more research, but it looks like this utility's obsolete.

An unrelated bug: clicking multiple times on the button invokes multiple instances of xvkbd.
